logstash-core-event-java/spec/event_spec.rb

```diff
@@ -135,4 +135,54 @@
       expect(event["tags"]).to eq(["foo"])
     end
   end
+
+  module DummyLogger
```

Comment:
I would move this to a support location and rename it somehow including the word stub/mock. Moving this to a support location will help reusing this loggers in case of others need it.